save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/Footer.kt

```diff
@@ -22,7 +22,7 @@ class Footer : RComponent<RProps, RState>() {
             div("container my-auto") {
                 div("copyright text-center my-auto") {
                     span {
-                        +"Copyright &copy; SAVE 2021"
+                        +"Copyright ${js("String.fromCharCode(169)")} SAVE 2021"
```

Comment:
Why it cann't be just string?

Reply:
It's an ugly workaround to display special characters like `©`. It should be working with HTML code `&copy;`, but for some reason it doesn't, and I've found this workaround.